core/amber/src/main/scala/edu/uci/ics/texera/web/auth/JwtAuth.scala

```diff
@@ -39,8 +39,9 @@ object JwtAuth {
   def jwtClaims(user: User, expireInDays: Int): JwtClaims = {
     val claims = new JwtClaims
     claims.setSubject(user.getName)
-    claims.setClaim("userId", user.getUid)
+    claims.setClaim("uid", user.getUid)
```

Comment:
Looks like there are other places that use "userId" in backend, like in `UserAuthenticator` and `ServletAwareConfigurator`. Please make sure to make it consistent, or revert this change if it is not necessary.

Reply:
Changed back to userId.